libs/repositories/src/datasources/coingecko.ts

```diff
@@ -171,7 +173,6 @@ export const COINGECKO_PLATFORMS: Record<number, string | undefined> = {
   [53935]: 'defi-kingdoms-blockchain',
   [5464]: 'saga',
   [5545]: 'duckchain',
-  [56]: 'binance-smart-chain',
```

Comment:
Why was this deleted?

Reply:
It was moved from the unsupported chains map into the supported chains one.